univariate-portfolio-sorts.qmd

```diff
@@ -102,27 +102,34 @@ The results indicate that we cannot reject the null hypothesis of average return
 
 ## Functional Programming for Portfolio Sorts
 
-Now we take portfolio sorts to the next level. We want to be able to sort stocks into an arbitrary number of portfolios. For this case, functional programming is very handy: we employ the [curly-curly](https://www.tidyverse.org/blog/2019/06/rlang-0-4-0/#a-simpler-interpolation-pattern-with-)-operator to give us flexibility concerning which variable to use for the sorting, denoted by `var`.\index{Curly-curly} We use `quantile()` to compute breakpoints for `n_portfolios`. Then, we assign portfolios to stocks using the `findInterval()` function. The output of the following function is a new column that contains the number of the portfolio to which a stock belongs.\index{Functional programming}
+Now we take portfolio sorts to the next level. We want to be able to sort stocks into an arbitrary number of portfolios. For this case, functional programming is very handy: we employ the [curly-curly](https://www.tidyverse.org/blog/2019/06/rlang-0-4-0/#a-simpler-interpolation-pattern-with-)-operator to give us flexibility concerning which variable to use for the sorting, denoted by `sorting_variable`.\index{Curly-curly} We use `quantile()` to compute breakpoints for `n_portfolios`. Then, we assign portfolios to stocks using the `findInterval()` function. The output of the following function is a new column that contains the number of the portfolio to which a stock belongs.\index{Functional programming} 
+
+In some applications, the variable used for the sorting might be clustered (e.g., at a lower bound of 0). Then, multiple breakpoints may be identical, leading to empty portfolios. Similarly, some portfolios might have a very small number of stocks at the beginning of the sample. Such portfolio imbalances can be easily identified in the output by checking the number of each portfolio's constituents over time. Additionally, consider investigating the breakpoints over time.
```

Comment:
Please remove this "easily identified" or clarify what could be analyzed here. Based on my understanding of the topic, I suggest something along the lines of: 
"Cases, where the number of portfolio constituents differs substantially due to the distribution of the characteristics, require careful consideration and, depending on the application, do prevent meaningful sorting mechanisms."

Reply:
Bin bei Stefan, dass ich hier warnender auftreten würde, aber leicht anders:
"Cases, where the number of portfolio constituents differs substantially due to the distribution of the characteristics, require careful consideration and, depending on the application, might require customized sorting approaches."

Reply:
Small alteration in Commit https://github.com/tidy-finance/website/pull/38/commits/9bcec97dee89f9c727ae15011d5da36c67db8e14